esm/streams/access-logs.js

```diff
@@ -89,7 +89,7 @@ export class ApplicationAccessLogStream extends CleverCloudSse {
   /**
    * shortcut for .on('APPLICATION_LOG', (event) => ...)
    * @param {Function} fn which handle logs
-   * @returns {any}
+   * @returns {ApplicationAccessLogStream}
```

Comment:
nitpick: you can do `@returns {this}` instead.

fix: can you move that into a separate commit?